lib/widgets/list.js

```diff
@@ -78,6 +78,10 @@ function List(options) {
   if (this.options.itemFocusEffects) {
     this.style.item.focus = this.options.itemFocusEffects;
   }
+  
+  if (this.options.itemHeight) {
+    this.style.item.height = 2;
```

Comment:
Shouldn't this be `this.style.item.height = this.options.itemHeight;`?
And when itemHeight is undefined, use 2?

Reply:
Absolutely; and if undefined, it should be `1` (which means no change)
@jrainville would you be interested in allowing `auto` value or `grow` to calculate how many lines it would take ? Based on screen length, input length and `\n` count in original strings, we can figure it out I guess